cartography/intel/aws/ec2.py

```diff
@@ -72,6 +84,32 @@ def get_ec2_vpcs(session, region):
     return client.describe_vpcs()
 
 
+def load_ec2_key_pairs(session, data, region, current_aws_account_id, aws_update_tag):
+    ingest_key_pair = """
+    MERGE (keypair:KeyPair:EC2KeyPair{keyname: {KeyName}, region: {Region}})
+    ON CREATE SET keypair.firstseen = timestamp()
+    SET keypair.keyfingerprint = {KeyFingerprint}, keypair.lastupdated = {aws_update_tag}
+    WITH keypair
+    MATCH (aa:AWSAccount{id: {AWS_ACCOUNT_ID}})
+    MERGE (aa)-[r:RESOURCE]->(keypair)
+    ON CREATE SET r.firstseen = timestamp()
+    SET r.lastupdated = {aws_update_tag}
+    """
+
+    for key_pair in data['KeyPairs']:
+        key_name = key_pair.get("KeyName", "")
```

Comment:
Since this code uses `KeyName` as an `id`, I don't think it would make sense to ingest a KeyPair that doesn't have a `KeyName`.  

For this reason, I think we should use `key_name = key_pair['KeyName']` so that the sync will crash (indicated bad data) if 'KeyName' is not present.